hbp_knowledge/adnp/van-dijck2019.bel

```diff
@@ -76,69 +76,98 @@ SET Citation = {"PubMed", "29724491"}
 SET Evidence = "Fifty-two percent of the individuals in this cohort present with
 severe intellectual disability at the age of assessment, 36% have a moderate
 disability and 12% have a mild disability. "
+SET Confidence = "High"
 p(HGNC:ADNP) -| path(MESH:"Intellectual Disability")
+UNSET Confidence
 
 SET Evidence = "Developmental delay is present in all individuals, with motor
 delay being one of the key features."
-p(HGNC:ADNP) -| path(MESH:"Developmental Disabilities")
-p(HGNC:ADNP) -> bp(GO:"motor behavior")
+SET Confidence = "Medium"
+p(HGNC:ADNP) -> path(MESH:"Developmental Disabilities")
+p(HGNC:ADNP) -| bp(GO:"motor behavior")
+UNSET Confidence
 
 SET Evidence = "Interestingly, individuals with a p.Tyr719* mutation start
 walking at 3.5 years, significantly later than the 2 years and 2 months of
 the remainder of the cohort (p<0.0001, One Way Anova)."
+SET Confidence = "High"
 p(HGNC:ADNP, frag("1_719")) -| bp(GO:"walking behavior")
+UNSET Confidence
 
+#not sure whether talking about mutated ADNP or non-mutated one
```

Comment:
If I remember correctly all children had mutations that lead to loss of the protein ADNP, therefore I coded everthing invers (see comment above)
you could also code it as variant, but I think coding the final result gives a more exact view on the function of the protein